src/test/unit_tests/test_nc_cli.js

```diff
@@ -991,16 +991,25 @@ mocha.describe('manage_nsfs cli', function() {
             assert_whitelist(config_data, config_options);
         });
 
-        mocha.it('cli whitelist ips is empty', async function() {
+        mocha.it('should fail - cli whitelist ips is empty', async function() {
             try {
                 await exec_manage_cli(type, '', { config_root, ips: '' });
                 assert.fail('should have failed with whitelist ips should not be empty.');
             } catch (err) {
-                assert_error(err, ManageCLIError.InvalidWhiteListIPFormat);
+                assert_error(err, ManageCLIError.UnsetArgumentIsInvalid);
             }
         });
 
-        mocha.it('cli whitelist formate is invalid', async function() {
+        mocha.it('cli whitelist ips is empty array', async function() {
+            const ips = [];
+            const res = await exec_manage_cli(type, '', { config_root, ips: JSON.stringify(ips) });
+            config_options.S3_SERVER_IP_WHITELIST = ips;
+            const config_data = await config_fs.get_config_json();
+            await assert_response('', type, res, ips);
+            assert_whitelist(config_data, config_options);
+        });
+
+        mocha.it('should fail - cli whitelist formate is invalid', async function() {
```

Comment:
We have a small typo (it was there before your changes)
```suggestion
        mocha.it('should fail - cli whitelist format is invalid', async function() {
```